style.css

```
@@ -0,0 +1,20 @@
+body {
+    background-color: green;
+    font-family: san-serif;
```

background and font-family do Not match
```suggestion
    background-image: url(........);
    font-family: 'Poppins' , san-serif;
```